compiler/rustc_builtin_macros/src/format.rs

```diff
@@ -195,12 +195,26 @@ fn make_format_args(
                                     Applicability::MaybeIncorrect,
                                 );
                             } else {
-                                let sugg_fmt = match args.explicit_args().len() {
-                                    0 => "{}".to_string(),
-                                    count => {
-                                        format!("{}{{}}", "{} ".repeat(count))
+                                let is_empty_tuple = |kind: &ExprKind| -> bool {
```

Comment:
Question: wait, this name seems misleading. The body of this closure matches on:

1. The actual empty tuple `()`, and
2. An empty block `{}` which is not a tuple?

Reply:
Done. Thank you.